src/dateUtils.js

```diff
@@ -0,0 +1,44 @@
+const { DateTime, Duration } = require('luxon');
+
+const toDate = val => {
+  if (DateTime.isDateTime(val)){
+    return val;
+  } else if (typeof val === 'number'){
+    return DateTime.fromMillis(val);
+  } else if (typeof val === 'string'){
+    if (val.match(/^[0-9]{4}-[0-9]{2}-[0-9]{2}$/)){
+      return DateTime.fromISO(val);
+    } else if (val.match(/^[0-9]{2}\/[0-9]{2}\/[0-9]{4}$/)){
+      return DateTime.fromFormat(val, 'dd/MM/yyyy');
```

Comment:
US use MM/dd/yyyy if I'm not mistaken. Date parsing is a big pain

Reply:
It would make sense to allow passing in a format string to evaluate against

Reply:
Rather than attempt to handle all types of formats, a user can create a DateTime and use it directly with setNow